src/three-components/TextureUtils.ts

```diff
@@ -38,6 +40,7 @@ Cache.enabled = true;
 const HDR_FILE_RE = /\.hdr$/;
 const ldrLoader = new TextureLoader();
 const hdrLoader = new RGBELoader();
+const cubemapSize = 256;
```

Comment:
Let's capitalize this e.g, `CUBEMAP_SIZE` (and yeah, `ldrLoader` / `hdrLoader` break our convention here a little; they should probably be static class members or something but we can fix that later).